airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/schemas/transactions.json

```diff
@@ -166,36 +200,40 @@
             "type": ["null", "object"],
             "properties": {
               "item_code": {
-                "type": ["null", "string"]
+                "type": ["null", "string"],
+                "maxLength": 1000
               },
               "item_name": {
-                "type": ["null", "string"]
+                "type": ["null", "string"],
+                "maxLength": 256
               },
               "item_description": {
                 "type": ["null", "string"]
               },
               "item_quantity": {
-                "type": ["null", "string"]
+                "type": ["null", "number"]
               },
               "item_unit_price": {
                 "type": ["null", "object"],
                 "properties": {
                   "currency_code": {
-                    "type": ["null", "string"]
+                    "type": ["null", "string"],
+                    "maxLength": 3
                   },
                   "value": {
-                    "type": ["null", "string"]
+                    "type": ["null", "number"]
                   }
                 }
               },
               "item_amount": {
                 "type": ["null", "object"],
                 "properties": {
                   "currency_code": {
```

Comment:
Please update the currency code and value properties according to the [documentation](https://developer.paypal.com/docs/api/transaction-search/v1/#definition-money).